lightningd/channel/channel.c

```diff
@@ -211,6 +225,9 @@ static struct io_plan *setup_peer_conn(struct io_conn *conn, struct peer *peer)
 
 static void peer_conn_broken(struct io_conn *conn, struct peer *peer)
 {
+	send_channel_update(peer, true);
```

Comment:
The alternative is to move this into the gossip daemon, allowing us to keep these concentrated on the two daemons, but it also means that we need to hand in an HSM socket to  `gossipd`.